lms/djangoapps/discussion/apps.py

```diff
@@ -12,7 +12,7 @@
 
 class DiscussionConfig(AppConfig):
     """
-    Application Configuration for Grades.
+    Application Configuration for Discussions.
```

Comment:
Haha!  This is why this docstring is superfluous - given the name of this class.